packages/react/src/components/TimePickerSelect/TimePickerSelect.js

```diff
@@ -16,6 +16,11 @@ const { prefix } = settings;
 
 export default class TimePickerSelect extends Component {
   static propTypes = {
+    /**
+     * Provide aria-label to the <select> element
+     */
+    ariaLabel: PropTypes.string,
```

Comment:
Does this need to be `aria-label` according to how it's used in `render`?

```suggestion
   ['aria-label']: PropTypes.string,
```